docs/source/gbd2019_models/causes/postpartum_depression/index.rst

```diff
@@ -0,0 +1,191 @@
+    - Note
+  * - PPD
+    - Postpartum depression
+    - 
+  * - MDD
+    - Major depressive disorder
+    - 
+
+Disease Overview
+----------------
+
+The postpartum period is a time of intense transition and can cause new mothers and birthing parents to be vulnerable to psychiatric disorders. Depression episodes can be twice as likely during the postpartum period relative to other times of life and postpartum depression (PPD) can adversely affect the wellbeing of mothers and birthing parents, infants, and other family members. 
+
+PPD refers to non-psychotic depressive episodes in the postpartum period that persist for more than two weeks and shares the same diagnostic criteria as major depressive disorder in the DSM-5 [Shorey-et-al-2018]_. Specifically, major depressive disorders are marked by depressed mood or loss of interest/pleasure that represent a change from the person's baseline and impaired functioning observed across social, occupational, and educational domains. Additional symptoms may include excessive sleeping or insomnia; change in eating, appetite, or weight; agitated or slow motor activity; fatigue; feeling worthless or inappropriately guilty; trouble concentrating; and repeated thoughts about death. [GBD-2019-Capstone-Appendix-PPD]_
+
+GBD 2019 Modeling Strategy
+--------------------------
+
+Postpartum depression is not specifically estimated by the GBD. Rather, we will inform the prevalence of postpartum depression from a recent review and meta-analysis performed by [Shorey-et-al-2018]_. This review focused on healthy mothers with no prior history of postpartum depression, but found similar results to previous evaluations of postpartum depression prevalence among the broader population. [Shorey-et-al-2018]_ reported that the incidence of postpartum depression was equal to 12% (95% CI 0.04–0.20).
+
+However, the GBD estimated that the average duration of major depressive disorder was 0.65 (95% UI: 0.59, 0.70) of one year (as informed through analysis of longitudinal studies on remission from major depressive disorder and the assumption that 40 years is the maximum duration of the condition), which we will use to inform the duration of postpartum depression in our simulation [GBD-2019-Capstone-Appendix-PPD]_ (page 1013). 
+
+Additionally, the GBD models severity distributions and associated disability weights for major depressive disorder, summarized in the table below [GBD-2019-Capstone-Appendix-PPD]_ (page 1013). Notably, the GBD defines major depressive disorder (MDD) as an episodic mood disorder involving the experience of one or more major depressive episode(s). Notably, the percent of case severity was determined from the US National Epidemiological Survey on Alchol and Related Conditions (conducted in two waves from 2001-2002 and 2004-2005) and the Australian National Survey of Mental Health and Wellbeing of Adults. Notably, burden due to major depressive disorder in GBD begins in the 1 to 4 year age group.
```

Comment:
```suggestion
Additionally, the GBD models severity distributions and associated disability weights for major depressive disorder, summarized in the table below [GBD-2019-Capstone-Appendix-PPD]_ (page 1013). Notably, the GBD defines major depressive disorder (MDD) as an episodic mood disorder involving the experience of one or more major depressive episode(s). The percent of case severity was determined from the US National Epidemiological Survey on Alchol and Related Conditions (conducted in two waves from 2001-2002 and 2004-2005) and the Australian National Survey of Mental Health and Wellbeing of Adults. Notably, burden due to major depressive disorder in GBD begins in the 1 to 4 year age group.
```

I think this is here to imply a limitation that I suggest we make more explicit (see below).